### ISSUE TITLE
Support spaces in test spec filter

### ISSUE THREAD

**Issue by ruudsieb** [philsquared/Catch#650]
It looks as if spaces are not allowed in the test spec filter on the command line.
If i execute this;

```
C:\develop\integrationtest\build\functional\Debug>functional_tests.exe "Open*" -l
Matching test cases:
  Open small VPV
  Open large VPV
  Open recent item 1
  Open recent item 3
4 matching test cases
```
Or this:
```
C:\develop\integrationtest\build\functional\Debug>functional_tests.exe "*recent*" -l
Matching test cases:
  Open recent item 1
  Open recent item 3
2 matching test cases
```
I get what i would expect, but in this case not;
```
C:\develop\integrationtest\build\functional\Debug>functional_tests.exe "*recent item*" -l
Matching test cases:
0 matching test cases
```
Or
```
C:\develop\integrationtest\build\functional\Debug>functional_tests.exe "Open recent item 1" -l
Matching test cases:
0 matching test cases
```

My usage scenario is; i just added a new testcase and want to test only that testcase, instead of running all cases. Yes, i could use a tag as a workaround, but above behaviour does not seem correct to me, or am i missing something? I'm using the generated main() by the way, so no custom command line parsing.

**Comment by philsquared** [comment]
It _should_ work exactly as you expect. If not it looks like a bug. I'll try and reproduce and.

**Comment by philsquared** [comment]
In fact I've just tried it out myself and it seems to work just fine for me.
Which version of Catch are you using?

**Comment by philsquared** [comment]
STOP PRESS!
The code base I just tested that out on was not on the latest Catch.
I just updated and can now reproduce your issue!
So it looks like the parser rework in Clara (the spun-out command line parser) is at fault. I'll fix it there. Thanks for reporting.

**Comment by ruderik1** [comment]
Glad it was a real issue, not misuse ;)

**Comment by philsquared** [comment]
Should be fixed now (v1.5.2)

**Comment by ruudsieb** [comment]
Fixed in 1.5.2
